doc/_themes/saltstack2/layout.html

```diff
@@ -2,10 +2,16 @@
 <!DOCTYPE html>
 {%- endblock %}
 
-{% set xxx = rellinks.extend([
+{%- set link_text = [] %}
+{%- for link_tuple in rellinks %}
+{%- set xxx = link_text.append(link_tuple[3]) %}
```

Comment:
```suggestion
{%- do link_text.append(link_tuple[3]) %}
```

If this doesn't work, it needs the [`do`](https://github.com/saltstack/salt/blob/v3006.2/tools/pre_commit.py#L89-L91) extension enabled. Don't know how we can enable that in Sphinx, or if that's enabled already, most likely, add `jinja2.ext.do` to the `extensions` config in [`conf.py`](https://github.com/saltstack/salt/blob/v3006.2/doc/conf.py#L144-L157)